### PR TITLE
ITSASU-1282, implement NewTabHelper

### PR DESCRIPTION
**Pull request by WozCapGem**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -563,11 +564,11 @@ affinity-group-error.agent.line1                                = To sign up for
 ##ADD MTDIT Overview
 
 mtdit-overview.heading                                          = Add Making Tax Digital for Income Tax to your business tax account
```
Comment (scullythornton1) [inline conf/messages:566]: row 34, 361, 371, 1065. still has (opens in new tab) text, please deleted them with Walse and also find where it has been used in the view.

File: app/views/individual/claimenrolment/AddMTDITOverview.scala.html [hunk 2]
```diff
@@ -32,16 +35,21 @@
 
 <h1 class="govuk-heading-l">@messages("mtdit-overview.heading")</h1>
 
-<p class="govuk-body"> @messages("mtdit-overview.paragraph1")
-    <a class="govuk-link" href="https://www.gov.uk/government/collections/making-tax-digital-for-income-tax" rel="noreferrer noopener" target="_blank">@messages("mtdit-overview.link_text")</a>
+<p class="govuk-body">
+    @Html(messages(
+    key = "mtdit-overview.paragraph1",
```
Comment (aug24) [inline app/views/individual/claimenrolment/AddMTDITOverview.scala.html:40]: This is definitely cleaner.  Nice

File: app/views/individual/claimenrolment/ClaimEnrolmentConfirmation.scala.html [hunk 3]
```diff
@@ -40,10 +43,11 @@ <h2 class="govuk-heading-m">@messages("claimConfirm.heading")</h2>
...
             <ol id="actionList" class="govuk-list govuk-list--number govuk-!-margin-2">
                 <li>
-                    <a class="govuk-link" id="compatibleSoftware" href="@appConfig.softwareUrl" rel="noreferrer noopener" target="_blank">
-                        @messages("claimConfirm.content.checklist.item1.link")
-                        @messages("claimConfirm.opens_in_a_new_tab")
-                    </a>
+
+                    @Html(messages(
+                    key = "base.new_tab_key",
+                    args = hmrcNewTabLinkHelper(NewTabLinkHelper(messages("claimConfirm.content.checklist.item1.link"), Some(appConfig.softwareUrl)))
```
Comment (aug24) [inline app/views/individual/claimenrolment/ClaimEnrolmentConfirmation.scala.html:49]: Fix indent?

File: conf/messages [hunk 4]
```diff
@@ -32,6 +32,7 @@ base.to                                                         = to
 base.caption                                                    = This section is
 base.submit_and_continue                                        = Submit and continue
 base.opens_in_a_new_tab                                         = (opens in new tab)
```
Comment (aug24) [inline conf/messages:34]: This hopefully can be removed now?

**Review by scullythornton1 — APPROVED** [review]

**Pull request merged**